### ISSUE TITLE
Thf dagger skill

### ISSUE THREAD

**Issue by oztralian** [LegionDark/Issues#766]
**Date & Time**: 3-23-16

**FFXI Client Version (use `/ver`)**: 

**Character Name**: Hotcarl

**Nation**: Bastok

**Job(level)/Sub Job(level)**: thf11/war5

**NPC or Monster or item Name**: 

**Zone name & Coordinates (use `@where`)**: 

**ffxiah.com link (for items issues only)**: 

**Multi-boxing? (multiple clients on same connection)**: No

**Steps To Reproduce / any other info**: level thf to 11. Sub war. Sword cap is 7 lvs higher than dagger. Dagger24 sword 31 eva 36. Skill caps are wrong.

**Comment by Samikana** [comment]
AFAIK this is a DSP issue.  Correct me if I'm wrong.

**Comment by oztralian** [comment]
Do i need to resubmit to dsp or is it already being handled? Im sorry if i submitted to the wrong area. Wasnt sure exactly were to post


Sent from my Sprint Samsung Galaxy Note5.-------- Original message --------From: Sami <notifications@github.com> Date: 3/23/2016  8:15 PM  (GMT-06:00) To: LegionDark/Issues <Issues@noreply.github.com> Cc: oztralian <ozzleelzzo@yahoo.com> Subject: Re: [Issues] Thf dagger skill (#766) 
AFAIK this is a DSP issue.  Correct me if I'm wrong.

**Comment by Samikana** [comment]
It doesn't hurt to submit it here, that way other players can see and be aware.  However, you might be suffering from a bug where a skill looks capped but it isn't.  Try unequipping and reeuipping your main weapon (and your shield if you have one on.) and it should resolve itself, as it doesn't seem there's any issue with the skill grades after checking them.  This still classifies as a DSP issue.

**Comment by Samikana** [comment]
![halsey_2016 03 23_220443](https://cloud.githubusercontent.com/assets/12781975/14006398/4968f398-f143-11e5-927d-5a562cf1e6ac.png)

**Comment by oztralian** [comment]
Tyvm for the quick replys. And the information about the bug. Much appreciated.


Sent from my Sprint Samsung Galaxy Note5.-------- Original message --------From: Sami <notifications@github.com> Date: 3/23/2016  9:03 PM  (GMT-06:00) To: LegionDark/Issues <Issues@noreply.github.com> Cc: oztralian <ozzleelzzo@yahoo.com> Subject: Re: [Issues] Thf dagger skill (#766) 
It doesn't hurt to submit it here, that way other players can see and be aware.  However, you might be suffering from a bug where a skill looks capped but it isn't.  Try unequipping and reeuipping your main weapon (and your shield if you have one on.) and it should resolve itself, as it doesn't seem there's any issue with the skill grades after checking them.  This still classifies as a DSP issue.

**Comment by TeoTwawki** [comment]
> AFAIK this is a DSP issue. Correct me if I'm wrong.

Yep, its a stupidly simple fix nobody here has time to do and test. Emphasis on test. Needs someoen to pull request to DSP and then LegionDark can get it with its DSP updates.
https://github.com/DarkstarProject/darkstar/issues/2380
https://github.com/DarkstarProject/darkstar/issues/1271
#557 
#290

**Comment by oztralian** [comment]
I unequipped my pile chain bought out of the server shop. It fixed the low cap. Happened when i lvd dnc as well. Dont know where to submit that but it seems it stopped my skills at the lvl i equipped it.

**Comment by TeoTwawki** [comment]
@oztralian https://github.com/DarkstarProject/darkstar/issues/2380 I think thats the same issue

So its 2 bugs really. DSP's skill stuff not matching retail, and DSP's cap getting set before it should be. From their issue report: "The workaround for it is to either zone or un/re-equip the weapon to reset it."

**Comment by Samikana** [comment]
So much derp.

**Comment by oztralian** [comment]
Just tryin to save some headscratching for people later.

**Comment by TeoTwawki** [comment]
@oztralian its cool. I _had_ a post I was leaving up for ppl, but it disappeared :confused:

**Comment by Samikana** [comment]
Let's keep this open and stickied so it's known.
